parser/src/cfg/mod.rs

```diff
@@ -444,6 +498,23 @@ pub fn parse_cfg_raw_string(
     }
     replace_custom_str_oscode_mapping(&local_keys.unwrap_or_default());
 
+    let cfg = root_exprs
```

Comment:
What's the reason for this change in order?

Reply:
`linux-unicode-u-code` can accept keys that are defined in localkeys